src/components/display/Chip.tsx

```diff
@@ -136,8 +136,8 @@ const ChipContainer = styled(Container)<{
 				return '100vh';
 		}
 	}};
-	cursor: ${({ onClick, onDoubleClick }): SimpleInterpolation =>
-		onClick || onDoubleClick ? 'pointer' : 'default'};
+	cursor: ${({ onClick, onDoubleClick, disabled }): SimpleInterpolation =>
+		disabled ? 'default' : (onClick || onDoubleClick) && 'pointer'};
```

Comment:
Instead of having a double conditional, why not just adding the disable condition to the first one?
`(onClick || onDoubleClick) && !disabled ? 'pointer' : 'default'`